npm/vite-dev-server/tsconfig.json

```diff
@@ -47,7 +47,7 @@
     "esModuleInterop": true,
     /** Allows us to strip internal types sourced from webpack */
     "stripInternal": true,
-    "importsNotUsedAsValues": "error",
```

Comment:
I think this is just oversight on my end. I need to add `"verbatimModuleSyntax": true` to get the expected behavior. good catch!

Reply:
I'm wondering if we will need to address this separately, since we cant use `"verbatimModuleSyntax": true` with `import` syntax inside common js modules. I wonder if we want to make an issue to reword the modules to leverage `"verbatimModuleSyntax": true` since there isn't anything that leads me to believe this is an issue?

Reply:
I don't think we have any polyfills or other content where this would be an issue but not entirely sure

Reply:
I wonder if we just want to pin to `~5.4.5` for the time being so we aren't subject to the breaking change in `5.5` with the removal of `"importsNotUsedAsValues": "error"`?

Reply:
@cacieprins I wound up going with the above comment in d3916e91f6